pwiz/analysis/demux/Jamfile.jam

```diff
@@ -52,9 +52,18 @@ lib pwiz_analysis_demux
         <library>$(PWIZ_ROOT_PATH)/pwiz/data/msdata//pwiz_data_msdata
     ;
 
+lib pwiz_analysis_demuxtestdata
```

Comment:
That's an impressive solution for your simulated test data. What's the advantage of it vs. trimming down real data? I've had the same thought several times over the years for simulating peptide DDA data, but every time I decided it would be easier to use some real data and filter it down to a small size.

Reply:
In this case collecting just one of either MSX overlap or multi-overlap data would take longer than writing the code that I did :) and I figured that once I wrote the bulk of the code to simulate a single permutation then it'd be quick to try and validate many other permutations of demuxing. I want to verify that the demuxing code is as general as I think I'm making it, where n-many overlaps can be demuxed (instead of only up to 2) and that MSX and n-many overlaps can be mixed. At this point collecting any one of these would either be a beast or just not possible with current instrument control software, so it came down to a decision of how to simulate data and whether the code should be internal to pwiz or if I should generate some simulated mzMLs externally.

I should note that in this branch, neither overlap MSX or n-many overlap are working (probably). Testing those would be the next step.

Reply:
I suppose when you want to test your code against data that is theoretically possible to acquire but nobody's actually done it, simulating it is definitely the next best thing. :100: :)